config/katello.migrations/202011084145-dont-user-pulpcore-deb-on-upgrades.rb

```diff
@@ -0,0 +1,11 @@
+KATELLO = 'katello'.freeze
+USE_PULP_2_FOR_DEB = 'use_pulp_2_for_deb'.freeze
+FOREMAN_PROXY_CONTENT = 'foreman_proxy_content'.freeze
+ENABLE_DEB = 'enable_deb'.freeze
+
+def upgrade
+  answers[KATELLO] = {} unless answers[KATELLO].is_a?(Hash)
```

Comment:
This line doesn't make sense in the current form. On line 11 it's already used as a hash so if it was `true` (which is technically a valid possibility), the migration would have crashed already.